### ISSUE TITLE
Attribute aria-hidden of div should not be put in div

### ISSUE THREAD

**Issue by mahui-cn** [kenwheeler/slick#4314]
> short description of the bug / issue, provide more detail below.

Chrome tell the errors as below:
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: div
Ancestor with aria-hidden:  <div class=​"text-center mx-2 slick-slide" data-slick-index=​"5" aria-hidden=​"true" tabindex=​"-1">​…​</div>​

It looks like the attribute aria-hidden of div should not be put here as the element div sometimes could be focused by user via mouse or keyboard which violate the latest rules of w3c specifications.

**Comment by mtibbits140** [comment]
I am also suddenly having an issue with this on a site where nothing has been changed. Any ideas on how to fix would be greatly appreciated!
